README.md

```diff
@@ -78,23 +78,23 @@ The code here is tested against [Kubernetes Cluster from Bluemix Container Servi
 
 ### 1. Set up environment
 
-Refer to the [instruction](https://cloud.google.com/bigquery/docs/reference/libraries) to install the client on your
-laptop to interact with Google BigQuery,   
+Refer to the [instructions](https://cloud.google.com/bigquery/docs/reference/libraries) to install the client on your
+laptop to interact with Google BigQuery:
 
 ```
-	pip install --upgrade google-cloud-bigquery
+pip install --upgrade google-cloud-bigquery
 ```
 
 Install the [Google Cloud SDK](https://cloud.google.com/sdk/docs/) on your laptop.  For example, on the Mac, download:
```

Comment:
should this step be ", .. download `google-cloud-sdk-168.0.0-darwin-x86_64.tar.gz` and unzip it.

then in the command part, do a `$ tar -xzf google-cloud-sdk-168.0.0-darwin-x86_64.tar.gz`...

Reply:
agree that the filename in a command block doesn't look right and it'll be better in the text.  I think we can skip the tar command though because some people just unzip by double-clicking on the download. Not sure if the tar command helps or not.